README.md

```diff
@@ -110,20 +110,25 @@ Using `sudo` use:
 
     $ sudo modprobe v4l2loopback
 
-This will create an additional video-device, e.g. /dev/video0 (the number
-depends on whether you already had video devices on your system), which can be
-fed by various programs.
+This will create an additional video-device, e.g. `/dev/video0` (the number
+depends on whether you already had video devices on your system). You can
+check which device is created by listing this directory contents.
+
+    $ ls -1 /sys/devices/virtual/video4linux
+    video0
+
+Various programs can then feed content to the loopback device `/dev/video0`.
+
```

Comment:
i think this is confusing: the loopback device can be `/dev/video0` or something else (i guess you refer to the `video0` in the previous line, but this is not clear.

Reply:
Is that better now?

Reply:
i still dont like the wording of the last sentence, as it fixates `/dev/video0`
i think the original wording was better in this respect.

How about simply adding the "You can check..." (until the example) after the current wording? (and add a note that the devices you see can be used under `/dev`).
e.g.

~~~
You can see which loopback devices are created by checking the `/sys/devices/virtual/video4linux` directory.
E.g. if there were two v4l2loopback devices `/dev/video0` and `/dev/video3` you would get:

    $ ls -1 /sys/devices/virtual/video4linux
    video0
    video3
~~~